src/lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.tsx

```diff
@@ -54,15 +55,15 @@ export function ClimateSettingScheduleForm({
 function Content({
   onBack,
 }: Omit<ClimateSettingScheduleFormProps, 'className'>): JSX.Element {
-  const { control, watch, resetField } = useForm({
+  const { control, watch } = useForm({
     defaultValues: {
       deviceId: '',
       name: '',
       startDate: '',
       endDate: '',
       timeZone: getSystemTimeZone(),
-      hvacModeSetting: 'heat_cool' as HvacModeSetting,
-      setPoints: {
+      climateSetting: {
+        hvacModeSetting: 'heat_cool' as HvacModeSetting,
```

Comment:
Still confused by the type casting. I'm mostly asking because I want to make sure the SDK types do not have problems. Typecasting like this indicates to me there is something we need to fix upstream in the SDK.
```suggestion
        hvacModeSetting: 'heat_cool',
```

Reply:
I think it's more an issue with react-hook-form, I will look again

Reply:
We could typecast `defaultValues` to `ClimateSettingScheduleFormFields` if that's preferable

Reply:
Passing the generic arg works 
<img width="691" alt="image" src="https://github.com/seamapi/react/assets/721372/de579c45-205d-4146-9cd9-fad40c14d75f">

Reply:
https://github.com/seamapi/react/pull/545/commits/24ceb055c0606202bdb65f94c683eed16b17b5fa#diff-4ce868582041afbebdf28cbf99e0256eea3422dda6a1e7c8e83a1cabfc8271e7R58